src/test/ui/parser/lex-bad-char-literals-5.rs

```diff
@@ -2,7 +2,6 @@
 // This test needs to the last one appearing in this file as it kills the parser
 static c: char =
     '\x10\x10'  //~ ERROR: character literal may only contain one codepoint
-                //~| ERROR: mismatched types
 ;
```

Comment:
can we add a type mismatch in the main body just to verify that the parser actually completes instead of bailing early?

Reply:
Do you mean you'd like me to add this?
```
fn main() {
    c = '\x10\x10'
}
```

Reply:
More like

```
fn main() {
    let c: &str = '\x10\x10';
}
```